.obs/dockerfile/micro-k3s-selinux-os/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+# Define the names/tags of the container
+#!BuildName: SL-Micro-k3s-selinux-container
+#!BuildTag: suse/sl-micro/%%SLMICRO_VERSION%%/k3s-selinux-os-container:latest
+#!BuildTag: suse/sl-micro/%%SLMICRO_VERSION%%/k3s-selinux-os-container:%VERSION%
+#!BuildTag: suse/sl-micro/%%SLMICRO_VERSION%%/k3s-selinux-os-container:%VERSION%-%RELEASE%
+#!BuildConstraint: hardware:disk:size unit=G 8
+#!UseOBSRepositories
+
+ARG SLMICRO_VERSION
+
+FROM suse/sl-micro/${SLMICRO_VERSION}/base-os-container:latest
+
+# dummy zypper call to get elemental into the build context and extract %VERSION% from it via _service
+RUN zypper in --no-recommends -y systemd-presets-branding-Elemental elemental
+
+# replace default kernel with smaller one, sufficient to run kvm guests
+RUN zypper in --no-recommends -y kernel-default-base -kernel-default qemu-guest-agent
+
+RUN zypper in --no-recommends -y container-selinux selinux-policy selinux-policy-targeted \
```

Comment:
I'd reduce it to `patterns-microos-selinux k3s-selinux`. This way we might also catch some eventual changes on Micro SELinux defaults pulled in by the pattern.